pkg/scheduler/framework/plugins/defaultpreemption/default_preemption_test.go

```diff
@@ -1263,6 +1256,9 @@ func TestSelectBestCandidate(t *testing.T) {
 			offset, numCandidates := pl.getOffsetAndNumCandidates(int32(len(nodeInfos)))
 			candidates, _ := dryRunPreemption(context.Background(), fwk, state, tt.pod, nodeInfos, nil, offset, numCandidates)
 			s := SelectCandidate(candidates)
+			if s == nil || len(s.Name()) == 0 {
+				return
+			}
```

Comment:
You can remove the first sub-test "No node needs preemption".

Reply:
Bump.

Reply:
deleted the test case.